### PR TITLE
[12.0] [MIG] partner_contact_personal_information_page

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Contacts' Personal Information Page
===================================

This module extends the contact management functionality.

Its only purpose is to serve as a base for other modules that add personal information fields to **contacts that are persons**.

Ultimately, this module is no longer required in version 9, as each module that was previously dependent upon it will declare what it needs for itself.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I don't like too much this module as it's a simple page in the notebook and I prefer to put the rest of the fields inside this page across the other pages or in the main section.

**Comment by Nikul-Chaudhary** [comment]
Yes, @pedrobaeza But I Just migrated module, wait for other reviewer answer.

If not any answer then I move to `internal_notes` pages or main section

**Review by andhit-r — APPROVED** [review]

File: partner_contact_personal_information_page/__manifest__.py
Comment (NL66278) [inline partner_contact_personal_information_page/__manifest__.py]: Do away with coding lines. And while you are on headers, change the licence lines to https:

**Comment by NL66278** [comment]
@pedrobaeza We do like this module, as it prevents an uncontrollable growth of notebook pages for any modules that has fields specifically relevant for persons.

**Comment by pedrobaeza** [comment]
OK, I think this information fits best in the popup window that appears in children contacts, as it's not intended for main partners.

**Comment by andhit-r** [comment]
@pedrobaeza IMO it can stay in the main form. A person with no company also need this page.

**Comment by rafaelbn** [comment]
Please @Nikul-Chaudhary Is it possible to have the new README structure?

File: partner_contact_personal_information_page/README.rst
Comment (cristinamartinrod) [inline partner_contact_personal_information_page/README.rst:41]: Minor changes. Should be v12

**Review by cristinamartinrod — APPROVED** [review]
LGTM 👍🏻

File: partner_contact_personal_information_page/README.rst
Comment (yajo) [reply partner_contact_personal_information_page/README.rst:41]: Yes, or use the readme generator to avoid this kind of maintenance in the future.

**Comment by NL66278** [comment]
@Nikul-Chaudhary I would be willing to merge this, if you take up the outstanding change requests.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @NL66278 Done

File: partner_contact_personal_information_page/readme/USAGE.rst
Comment (yajo) [inline partner_contact_personal_information_page/readme/USAGE.rst]: From here to the end of the file is boilerplate. Remove it please.

**Review by yajo — CHANGES_REQUESTED** [review]

File: partner_contact_personal_information_page/readme/USAGE.rst
Comment (Nikul-Chaudhary) [reply partner_contact_personal_information_page/readme/USAGE.rst]: @Yajo Done

**Review by yajo — APPROVED** [review]

**Comment by MarkingSC** [comment]
I proposed that issue, #745 
I forket it and now I got the contact fields that I mentioned in the issue description. 
But I still don't know who should have accesss to that information. 
What you think about it? 
Screenshot_20190527_123826

**Pull request merged**
